### DISCUSSION TITLE
Make bot in voice channel all the time

### DISCUSSION THREAD

**Discussion by Tarekkhayrt** [eritislami/evobot#293] [Q&A]
Hello, can anyone know how can i make the bot stays in the voice channel all the time?
also how to make the bot can't follow any order from any other room than "Music room" for example?

**Comment by Tarekkhayrt** [comment]
Ok i figured it out
If anyone's interested this is the way:
In the file "play.js" in "include" folder
You can comment the line 20 to make the bot can't leave the voice channel room after the song finished.
Thanks for nothing guys

**Reply by Mati2181** [reply]
HI, 
what do I have to put so that the bot does not leave the voice channel?

**Reply by Prasham27** [reply]
@Tarekkhayrt I'm interested in doing this how can I do this?
